third-party/cmake/BuildLuarocks.cmake

```diff
@@ -80,24 +134,25 @@ if(USE_BUNDLED_BUSTED)
     ARGS build luasocket 3.0rc1-2 ${LUAROCKS_BUILDARGS}
     COMMAND ${LUAROCKS_BINARY}
     ARGS build xml 1.1.2-1 ${LUAROCKS_BUILDARGS}
-    COMMAND touch ${DEPS_LIB_DIR}/luarocks/rocks/stable-busted-deps
+    COMMAND touch ${HOSTDEPS_LIB_DIR}/luarocks/rocks/stable-busted-deps
     DEPENDS lpeg)
   add_custom_target(stable-busted-deps
-    DEPENDS ${DEPS_LIB_DIR}/luarocks/rocks/stable-busted-deps)
-
-  add_custom_command(OUTPUT ${DEPS_BIN_DIR}/busted
+    DEPENDS ${HOSTDEPS_LIB_DIR}/luarocks/rocks/stable-busted-deps)
+ 
+  add_custom_command(OUTPUT ${HOSTDEPS_BIN_DIR}/busted
     COMMAND ${LUAROCKS_BINARY}
     ARGS build https://raw.githubusercontent.com/Olivine-Labs/busted/v2.0.rc8-0/busted-2.0.rc8-0.rockspec ${LUAROCKS_BUILDARGS}
     DEPENDS stable-busted-deps)
   add_custom_target(busted
-    DEPENDS ${DEPS_BIN_DIR}/busted)
-
-  add_custom_command(OUTPUT ${DEPS_LIB_DIR}/luarocks/rocks/nvim-client
+    DEPENDS ${HOSTDEPS_BIN_DIR}/busted)
+ 
+  add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/nvim-client
     COMMAND ${LUAROCKS_BINARY}
-    ARGS build https://raw.githubusercontent.com/neovim/lua-client/0.0.1-12/nvim-client-0.0.1-12.rockspec ${LUAROCKS_BUILDARGS} LIBUV_DIR=${DEPS_INSTALL_DIR}
+    ARGS build https://raw.githubusercontent.com/neovim/lua-client/0.0.1-12/nvim-client-0.0.1-12.rockspec ${LUAROCKS_BUILDARGS} LIBUV_DIR=${HOSTDEPS_INSTALL_DIR}
     DEPENDS busted libuv)
   add_custom_target(nvim-client
-    DEPENDS ${DEPS_LIB_DIR}/luarocks/rocks/nvim-client)
-
+    DEPENDS ${HOSTDEPS_LIB_DIR}/luarocks/rocks/nvim-client)
+ 
```

Comment:
@equalsraf you can find whitespace issues using `git log --check -9`. The `--check` flag can also be used for the `show` and `diff` sub-commands.